crates/bevy_asset/src/server/mod.rs

```diff
@@ -913,29 +910,19 @@ impl AssetServer {
                     AssetServerMode::Processed { .. } => match source.processed_reader() {
                         Ok(reader) => reader,
                         Err(_) => {
-                            error!(
-                                "Failed to load {path}. AssetSource {:?} does not have a processed AssetReader",
-                                path.source()
-                            );
+                            error!("Failed to load {path}. AssetSource {:?} does not have a processed AssetReader", path.source());
                             return;
                         }
                     },
                 };
 
                 let mut handles = Vec::new();
                 match load_folder(source.id(), path.path(), asset_reader, &server, &mut handles).await {
-                    Ok(_) => server.send_asset_event(InternalAssetEvent::Loaded {
-                        id,
-                        loaded_asset: LoadedAsset::new_with_dependencies(
-                            LoadedFolder { handles },
-                            None,
-                        )
-                        .into(),
-                    }),
+                    Ok(_) => server.send_asset_event(InternalAssetEvent::Loaded { id, loaded_asset: LoadedAsset::new_with_dependencies(LoadedFolder { handles }, None).into() }),
```

Comment:
This is one of the changes I'm talking about that aren't an improvement at all and just seem like a rustfmt failure.

Reply:
I guess we'll have to manually format these lines. If there aren't too many, maybe we can just catch them in review.

Reply:
My issue is that it's most of the changes that suffer from this, I only pointed out the first one that were too bad to ignore for me, but the more I scroll the more I see changes that I see as complete downsides.